src/Microsoft.ApplicationInsights.AspNet/TelemetryInitializers/DomainNameRoleInstanceTelemetryInitializer.cs

```diff
@@ -5,31 +5,25 @@
     using System.Net;
     using System.Net.NetworkInformation;
     using System.Threading;
-
+    using Channel;
     using Microsoft.ApplicationInsights.DataContracts;
-    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.AspNet.Http;
+    using TelemetryInitializers;
 
-    /// <summary>
-    /// A telemetry context initializer that populates device context role instance.
```

Comment:
have you removed comment completely? Please return back if you did - it is a public class